### PR TITLE
Feat: Add sns aggregator to deploy script

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Easily deploy SNS Aggregator to a testnet.

# Changes

* Support new argument in `./deploy.sh` `--sns-aggregator` that will deploy the aggregator to the network.

# Tests

Not applicable.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -66,12 +70,43 @@ if [[ "$DEPLOY_II" == "true" ]]; then
...
+      return
+    fi
+  done
+  echo "null"
+}
+
+static_host() {
+  first_not_null \
+    "$(jq -re '.networks[env.DFX_NETWORK].config.STATIC_HOST' dfx.json)" \
+    "$(jq -re '.networks[env.DFX_NETWORK].config.HOST' dfx.json)"
```
Comment (bitdivine) [inline deploy.sh:88]: The equivalent of Javascript's || in jq is //:
```suggestion
    jq -re '.networks[env.DFX_NETWORK].config | .STATIC_HOST // .HOST' dfx.json
```
Explanation:
* Get the config.  ` | ` then pipes the result to the next command.
* Get `.STATIC_HOST` or, if that is null, `.HOST`, from the config.  If neither is found, the output will be null.  You could add a different with e.g. ` // ""` for an empty string as default if you want.

Demo:
```
max@sinkpad:~/dfn/nns-dapp/branches/no-mock-conversion-rate (6:57)$ DFX_NETWORK=local jq -re '.networks[env.DFX_NETWORK].config| .STATIC_HOST // .HOST' dfx.json
http://localhost:8080
max@sinkpad:~/dfn/nns-dapp/branches/no-mock-conversion-rate (7:21)$ DFX_NETWORK=mainnet jq -re '.networks[env.DFX_NETWORK].config| .STATIC_HOST // .HOST' dfx.json
https://icp0.io
max@sinkpad:~/dfn/nns-dapp/branches/no-mock-conversion-rate (7:33)$ DFX_NETWORK=askdjhadsjk jq -re '.networks[env.DFX_NETWORK].config| .STATIC_HOST // .HOST' dfx.json
null
```

**Review by bitdivine — APPROVED** [review]
LGTM, thanks.  The code can be simplified a bit, if you want, but it's up to you.

**Review by bitdivine — APPROVED** [review]
LGTM, thank you.  Code can be simplified a bit if you want.

File: deploy.sh (hunk 1 above)
Comment (lmuntaner) [reply deploy.sh:88]: Cool! thanks!

**Pull request merged**